tests/test_production.py

```diff
@@ -172,16 +172,13 @@ async def get_context(config_override=None):
         "cot_product": "mobile",
         "check_task": False,  # These tasks run on level t workers.
     },
-    pytest.param(
-        {
-            "name": "adhoc-signing",
-            "taskcluster_root_url": "https://firefox-ci-tc.services.mozilla.com/",
-            "index": "adhoc.v2.adhoc-signing.bug1799220.release-signing.latest",
-            "task_type": "signing",
-            "cot_product": "adhoc",
-        },
-        marks=pytest.mark.xfail,
-    ),
+    {
+        "name": "adhoc-signing",
+        "taskcluster_root_url": "https://firefox-ci-tc.services.mozilla.com/",
+        "index": "adhoc.v2.adhoc-signing.latest.taskgraph.decision",
+        "task_type": "decision",
+        "cot_product": "adhoc",
+    },
```

Comment:
Could you add this test in addition to the old one? The old one is going to test the full chain from signing -> docker-image -> decision, while the latter will only test decision. So the old test is still providing coverage that the new one is not.

Reply:
Unfortunately we won't be able to use an index. Should we skip testing, or point to a specific manifest?

Reply:
Why can't we use the index? Once we adhoc sign something we can just update the index to point to whatever manifest was adhoc signed and remove the `xfail`. It doesn't have to be for this manifest specifically, it can be for any of them.